docs/source/docker/configuration/configuration.rst

```diff
@@ -24,71 +24,116 @@ The following are parameters which can be passed to the container:
-    show_video_sampling_timeline: True
+  # Access token, get it from app.lightly.ai.
+  token: ''
+
+  # If specified, the docker is registered as a worker on the Lightly platform.
+  worker_id: ''
+
+  # Set to True to check whether installation was successful.
+  sanity_check: False 
+
+  # Set to the path to a relevant_filenames.txt to run the docker on a subset of 
+  # filenames. The docker will ignore all files in the input directory not listed 
+  # here. Each filename in the must be in a separate line and relative to the 
+  # input directory. The path relevant_filenames_file to the relevant_filenames.txt 
+  # is relative to the shared directory.
+  relevant_filenames_file: ''
+
+  # Set to False to disable check for corrupted images.
+  enable_corruptness_check: True
+  corruptness_check:
+    # Threshold in [0, 1] which determines the sensibility of the corruptness check
+    # for video frames. Every frame which has an internally computed corruptness
+    # score larger than the specified threshold will be classified as corrupted.
+    corruption_threshold: 0.1
+
+  # Remove exact duplicates.
+  remove_exact_duplicates: True
+
+  # Path to the checkpoint relative to the shared directory.
+  checkpoint: ''
+
+  # Path to the embeddings file relative to the shared directory.
+  embeddings: ''
+
+  # Enable training, only possible when no embeddings are passed.
+  enable_training: False
+
+  # Dump the final dataset to the output directory.
+  dump_dataset: False
+  dump_sampled_embeddings: True
+  # Set the size of the dumped images, use =x or =[height,width] to match the 
+  # shortest edge to x or to resize the image to (height, width), use =-1 for no 
+  # resizing (default). This only affects the output size of the images dumped to 
+  # the output folder with dump_dataset=True. To change the size of images 
+  # uploaded to the lightly platform or your cloud bucket please use the 
+  # lightly.resize option instead.
+  output_image_size: -1
+  output_image_format: 'png'
+
+  # Upload the dataset to the Lightly platform.
+  upload_dataset: False
+
+  # pretagging
+  pretagging: False
+  pretagging_debug: False
+  pretagging_config: ''
+
+  # Append weak labels.
+  append_weak_labels: False
+
+  # Normalize the embeddings to unit length.
+  normalize_embeddings: True
+
+  # active learning scorer
+  scorer: 'object-frequency'
+  scorer_config:
+    frequency_penalty: 0.25
+    min_score: 0.9
+
+
+  # sampling
+  # Choose from ['coreset', 'random'].
+  method: 'coreset'
+  stopping_condition:
+    # Float in [0., 1.] for percentage, int for number of samples, -1 means inactive.
+    n_samples: -1    
+      # Float, minimum distance between two images in the sampled dataset, -1. means inactive.           
+    min_distance: -1.
+  selected_sequence_length: 1
+
+  # datapool
+  datapool:
+    # Name of the datapool.
+    name:
+    # If True keeps backup of all previous data pool states.
+    keep_history: True
+    id:
```

Comment:
You should comment on the "id"

Reply:
Added it. Please verify that the content is correct.